lineman/app/models/group_model.coffee

```diff
@@ -17,6 +17,13 @@ angular.module('loomioApp').factory 'GroupModel', (BaseModel, AppConfig) ->
       @hasMany 'subgroups', from: 'groups', with: 'parentId', of: 'id'
       @belongsTo 'parent', from: 'groups'
 
+    proposals: ->
```

Comment:
yea.. you're both right. Hannah and I discussed either doing a manual hasManyThrough or putting group_id in to the serializer. 

I worried that groupId might cause n+1 lookups so thought we should try this... the caveat was that ended up hasOneing discussion into the motionSerializer anyway.. so reflecting on this we could have also gone the other way... ie: it's worth trying that out @gdpelican...